test/ErrorRecovery/SubsequentDecls.C

```diff
@@ -24,7 +24,10 @@ namespace test { int y = 0; }
 .compareState "testSubsequentDecls"
 // CHECK-NOT: Differences
 
-TemplatedF((int)2) // expected-diagnostics{{C++ requires a type specifier for all declarations}} expected-diagnostics{{expected ';' after top level declarator}}
+TemplatedF((int)2)
+// CHECK: IncrementalExecutor::executeFunction: symbol '{{.*}}' unresolved while linking [cling interface function]!
```

Comment:
What happened to the expected diag? I'd still expect that diag - at least on other platforms? That should not change due to your patch?